spec/lib/traject/config/callnum_facet_spec.rb

```diff
@@ -60,12 +60,6 @@ def record_with_999(call_number:, scheme:, indexer:, home_location: 'STACKS', li
       expect(record_with_999(call_number: 'M123 .M234', scheme: 'AUTO', indexer:)[field]).to be_nil
     end
 
-    it 'handles skipped items (by not including them)' do
```

Comment:
Do we want to fully delete this test or just replace the loc with one of the included ones, like `SHADOW` ?

Reply:
Yeah, good idea. I've added a test for SHADOW.